app/Resources/views/row/banner_slider.html.twig

```diff
@@ -0,0 +1,67 @@
+<figure class="cd-image-container">
+    {{ ez_render_field(content, 'right_image', {
+        template: 'fields/slider_image.html.twig',
+        parameters: {
+            class: 'img-responsive'
+        }
+    }) }}
+
+    <div class="cd-resize-img">
+        {{ ez_render_field(content, 'left_image', {
+            template: 'fields/slider_image.html.twig'
+        }) }}
+    </div>
+
+    <span class="cd-handle"></span>
+    <h1 class="ez-platform-label">eZ Platform</h1>
+    <a class="btn btn-primary download" href="#Download">DOWNLOAD EZ PLATFORM</a>
+</figure>
+
+<script>
+    {% autoescape 'js' %}
```

Comment:
This script is only for the slider. I don't want it on all the page of the website